ros/angel_system_nodes/angel_system_nodes/hl2ss_ros_bridge.py

```diff
@@ -161,6 +194,8 @@ def __init__(self):
         log.info("SI client connected!")
         self.connect_hl2ss_audio()
         log.info("Audio client connected!")
+        self.connect_hl2ss_sm()
+        log.info("SM client connected!")
```

Comment:
Do you know what the behavior is if the HL2SS server does not have the Spatial-mapping functionality enabled? My concern is compatibility with the BBN system which I believe is not outputting all of the possible streams (or is fully up to date). If this does not gracefully fail, we will likely need to add some ability here to "disable" trying to connect to various servers (e.g. if "disable" is passed in as the "topic", don't create a client, etc.).

Reply:
This is added now, so if you pass "disable" as the topic name for any of the sensor streams, they will be disabled.